scheduler/kubernetes/install.md

```diff
@@ -110,15 +110,19 @@ If you are still experiencing issues, please refer to [Troubleshooting PX on Kub
 
 
 #### Restricting PX to certain nodes
-To restrict Portworx to install on only a subset of nodes in the Kubernetes cluster, please set the `px/enabled=false` Kubernetes label on the minion nodes you _do not_ wish to install Portworx on.  For example, to prevent Portworx from installing and starting on _minion2_ and _minion5_ nodes, run the following:
+To restrict Portworx to run on only a subset of nodes in the Kubernetes cluster, we can use the `px/enabled` Kubernetes label on the minion nodes you _do not_ wish to install Portworx on.  Below are examples to prevent Portworx from installing and starting on _minion2_ and _minion5_ nodes.
 
-```
-$ kubectl label nodes minion2 "px/enabled=false" --overwrite
-$ kubectl label nodes minion5 "px/enabled=false" --overwrite
-```
-
->**Note:**<br/> If you are using the PX-OCI containers, and the PX-OCI has already been deployed on your nodes, please use the `px/enabled=remove` label instead. <br/>Note that with PX-OCI containers, the `px/enabled=false` removes only the Portworx "oci-monitor" component, while retaining PX-OCI service. Applying the `px/enabled=remove` label will uninstall both PX-OCI service, and the "oci-monitor".
+If Portworx Daemonset is not yet deployed in your cluster:
+  ```bash
+  $ kubectl label nodes minion2 minion5 px/enabled=false --overwrite
+  ```
 
+If Portworx has already been deployed in your cluster:
+  ```bash
+  $ kubectl label nodes minion2 minion5 px/enabled=remove --overwrite
```

Comment:
I don't think this will not render correctly for bash (it's not a valid bash-script since it starts w/ `$`).
I suggest you remove the `"""bash` and use plain `"""` - this line does not use strings/numbers/etc, there's nothing to color differently.

Reply:
Fixed in incremental.